recipes/singleton-decorator/LICENSE

```diff
@@ -0,0 +1,219 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
The license mentioned in the metadata is not the same. This is GPL whereas the recipe states AGPL.

Reply:
Good catch. I have updated the license.